kubernetes/table_kubernetes_event.go

```diff
@@ -0,0 +1,200 @@
+package kubernetes
+
+import (
+	"context"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+func tableKubernetesEvent(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "kubernetes_event",
+		Description: "Kubernetes Event is a report of an event somewhere in the cluster.",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "namespace"}),
+			Hydrate:    getK8sEvent,
+		},
+		List: &plugin.ListConfig{
+			Hydrate:    listK8sEvents,
+			KeyColumns: getCommonOptionalKeyQuals(),
+		},
+		Columns: k8sCommonColumns([]*plugin.Column{
+			{
+				Name:        "action",
+				Type:        proto.ColumnType_STRING,
+				Description: "What action was taken/failed with the regarding object.",
+				Transform:   transform.FromField("Action"),
```

Comment:
We need to use FromField when the output parameters are not in the Camel case, or we need to extract the value from an object. 

For example, if the output parameters are like `Action`, `ActionTaken` etc, then we do not need to use FromField.  

But if the output parameters are like `ID` or the value needs to extract from an object like `Spec.Status`, then we need to use FromField.

Reply:
Thanks, I think I have removed the unnecessary ones.